<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Groupbox */

QGroupBox {
    border: 1px solid #DDD;
    border-radius: 9px;
    margin-top: 9px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QGroupBox" name="OptionsGB">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>220</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Options</string>
      </property>
      <widget class="QSpinBox" name="SubdivSteps">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
      <widget class="QLabel" name="SubdivLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Loop subdivision steps</string>
       </property>
      </widget>
      <widget class="QLabel" name="RotateLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Rotation around Y-axis</string>
       </property>
      </widget>
      <widget class="QDial" name="RotateDial">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>50</width>
         <height>64</height>
        </rect>
       </property>
       <property name="maximum">
        <number>359</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="notchTarget">
        <double>12.000000000000000</double>
       </property>
       <property name="notchesVisible">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="LoadOBJ">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Load OBJ file</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="MainView" name="MainDisplay"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainView</class>
   <extends>QOpenGLWidget</extends>
   <header>mainview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
